src/app/components/core/dashboard/dashboard.component.ts

```diff
@@ -224,6 +224,9 @@ export class DashboardComponent implements OnInit, OnDestroy {
             }
           });
         });
+        if (this.isAlive === false) {
+          this.isAlive = true;
+        }
```

Comment:
if we have to set it to true, then if check can be avoided; as it can have only true / false and we want to set true anyway

Reply:
This change is not required!